src/OrleansRuntime/ReminderService/LocalReminderService.cs

```diff
@@ -201,17 +203,28 @@ public Task Stop()
         private async Task ReadAndUpdateReminders()
         {
             if (stoppedCancellationTokenSource.IsCancellationRequested) return;
+            localRangeSequence++;
+            var cachedLocalRangeSequence = localRangeSequence;
```

Comment:
yep, I wasn't sure about naming, so went with the same naming convention. I'll rename to make the distinction more obvious.